src/runtime/gc_conservative.go

```diff
@@ -240,7 +240,7 @@ func calculateHeapAddresses() {
 	totalSize := heapEnd - heapStart
 
 	// Allocate some memory to keep 2 bits of information about every block.
-	metadataSize := totalSize / (blocksPerStateByte * bytesPerBlock)
+	metadataSize := align((totalSize / (blocksPerStateByte * bytesPerBlock)) + 1)
```

Comment:
I do not think that this is correct. First, `align` works differently on different targets depending on their required alignment. . .  which is not what we need here. Second, the inner expression isn't rounding down, it is rounding up and then adding 1.

If we want to round up, I think this should be:
```suggestion
	metadataSize := (totalSize + (blocksPerStateByte * bytesPerBlock)) / (blocksPerStateByte * bytesPerBlock)
```